index.html

```diff
@@ -61,9 +61,17 @@ <h1 class="display-3">Bikes for Refugees</h1>
 							<a class="btn btn-secondary btn-lrg" href="#">Volunteer</a>
 						</div>
 					</div>
+					<div class="container row justify-content-around">
```

Comment:
You cannot use a container and a row on the same level. The order must be: 
1) `container` (grandparent)
2) `row` (parent)
3) `col-...` (children) **note: must add up to 12**